### PR TITLE
CPM-148: add guidelines in attributes API

### PR DESCRIPTION
**Pull request by nmarniesse**

### CODE REVIEW

File: content/php-client/resources/catalog-structure/attributes.md [hunk 1]
```diff
@@ -36,7 +36,11 @@ $client = new \Akeneo\Pim\ApiClient\AkeneoPimClientBuilder('http://akeneo.com/')
  *     'scopable'               => false,
  *     'labels'                 => [
  *         'en_US' => 'Sale date',
- *         'fr_FR' => 'Date des soldes'],
+ *         'fr_FR' => 'Date des soldes',
+ *     ],
+ *     'descriptions'           => [
+ *         'en_US' => 'The release date for the summer sale 2017',
+ *         'fr_FR' => 'Date des soldes pour lété 2017',
```
Comment (pierallard) [inline content/php-client/resources/catalog-structure/attributes.md:43]: l\'été ?

File: content/php-client/resources/catalog-structure/attributes.md [hunk 2]
```diff
@@ -108,9 +112,13 @@ $client->getAttributeApi()->create('release_date', [
...
     'labels'                 => [
-        'en_US': 'Sale date',
-        'fr_FR': 'Date des soldes'],
-     ]
+        'en_US' => 'Sale date',
+        'fr_FR' => 'Date des soldes',
+    ],
+    'descriptions'           => [
+        'en_US' => 'The release date for the summer sale 2017',
+        'fr_FR' => 'Date des soldes pour lété 2017',
```
Comment (pierallard) [inline content/php-client/resources/catalog-structure/attributes.md:120]: same

File: content/php-client/resources/catalog-structure/attributes.md [hunk 3]
```diff
@@ -148,9 +156,13 @@ $client->getAttributeApi()->upsert('release_date', [
...
```
Comment (pierallard) [inline content/php-client/resources/catalog-structure/attributes.md:164]: (there are other ones)

**Review by pierallard — APPROVED** [review]

**Review by mmetayer — APPROVED** [review]

**Comment by meganelepalud-akeneo** [comment]
Hello! :) Is it ready to be published?

**Comment by nmarniesse** [comment]
> Hello! :) Is it ready to be published?

Hello @meganelepalud-akeneo we have a functional issue in the fixture. We hope find a solution in the next few days

**Comment by meganelepalud-akeneo** [comment]
> Hello @meganelepalud-akeneo we have a functional issue in the fixture. We hope find a solution in the next few days

Ok, thanks. Just wanted to be sure, we don't forget anything already release ;)

**Pull request merged**